---
noteId: "3403dfd0323511f0b9c70d17743ef7b7"
tags: []

---

> üõë This file is source code, not the primary documentation location! üõë
>
> See **https://typescript-eslint.io/rules/your-rule-name** for documentation.

## Examples

To fill out: tell us more about this rule.

<!--tabs-->

### ‚ùå Incorrect

```ts
// To fill out: incorrect code
```

### ‚úÖ Correct

```ts
// To fill out: correct code
```

## When Not To Use It

To fill out: why wouldn't you want to use this rule?
For example if this rule requires a feature released in a certain TS version.
